adyenv6core/src/com/adyen/v6/facades/DefaultAdyenCheckoutFacade.java

```diff
@@ -401,7 +401,7 @@ public OrderData authorisePayment(final HttpServletRequest request, final CartDa
         updateCartWithSessionData(cartData);
         String adyenPaymentMethod = cartData.getAdyenPaymentMethod();
 
-        if (adyenPaymentMethod.equals(PAYMENT_METHOD_BOLETO) || adyenPaymentMethod.equals(PAYPAL_ECS) || adyenPaymentMethod.startsWith(RATEPAY)) {
+        if (adyenPaymentMethod.equals(PAYPAL_ECS) || adyenPaymentMethod.startsWith(RATEPAY)) {
```

Comment:
same check as canUseAPI() can we combine these two a method that can be called from those two places.

Reply:
We are planning to move RATEPAY to checkout soon. So then we will have only PAYPAL_ECS left. Thats why I am not doing any optimizations on old code.

Reply:
clear.